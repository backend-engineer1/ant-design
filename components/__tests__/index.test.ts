import * as antd from '..';

const OLD_NODE_ENV = process.env.NODE_ENV;
process.env.NODE_ENV = 'development';

describe('antd', () => {
  afterAll(() => {
    process.env.NODE_ENV = OLD_NODE_ENV;
  });

  it('exports modules correctly', () => {
    expect(Object.keys(antd)).toMatchSnapshot();
  });
});
