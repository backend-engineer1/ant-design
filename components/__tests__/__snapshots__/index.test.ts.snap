// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`antd > exports modules correctly 1`] = `
[
  "Affix",
  "Alert",
  "Anchor",
  "App",
  "AutoComplete",
  "Avatar",
  "BackTop",
  "Badge",
  "Breadcrumb",
  "Button",
  "Calendar",
  "Card",
  "Carousel",
  "Cascader",
  "Checkbox",
  "Col",
  "Collapse",
  "ColorPicker",
  "ConfigProvider",
  "DatePicker",
  "Descriptions",
  "Divider",
  "Drawer",
  "Dropdown",
  "Empty",
  "FloatButton",
  "Form",
  "Grid",
  "Image",
  "Input",
  "InputNumber",
  "Layout",
  "List",
  "Mentions",
  "Menu",
  "message",
  "Modal",
  "notification",
  "Pagination",
  "Popconfirm",
  "Popover",
  "Progress",
  "QRCode",
  "Radio",
  "Rate",
  "Result",
  "Row",
  "Segmented",
  "Select",
  "Skeleton",
  "Slider",
  "Space",
  "Spin",
  "Statistic",
  "Steps",
  "Switch",
  "Table",
  "Tabs",
  "Tag",
  "theme",
  "TimePicker",
  "Timeline",
  "Tooltip",
  "Tour",
  "Transfer",
  "Tree",
  "TreeSelect",
  "Typography",
  "Upload",
  "version",
  "Watermark",
]
`;
